src/main/java/racing/TrialNum.java

```diff
@@ -0,0 +1,16 @@
+package racing;
+
+public class TrialNum {
```

Comment:
객체지향 생활 체조 원칙의 `규칙 5: 줄여쓰지 않는다(축약 금지).`에 따라, 
이름을 약어나 축약형 대신 완전한 단어로 표현해보면 어떨까요?
```suggestion
public class TrialNumber {
```